rcgen/Cargo.toml

```diff
@@ -36,7 +36,7 @@ x509-parser = { workspace = true, features = ["verify"], optional = true }
 zeroize = { version = "1.2", optional = true }
 
 [features]
-default = ["crypto", "pem", "ring"]
+default = ["crypto", "pem", "ring", "x509-parser"]
```

Comment:
I don't think this what we want? It looks like this might be only for (some of) the tests? If so, we can make x509-parser a required dev-dependency.

Reply:
Oh I think I accidentally commited that part. It was so my ide would actually be helpful for the section I was working on.

Reply:
I suggest running your IDE with --all-features (that's what I do).